helm/cluster/values.schema.json

```diff
@@ -1544,37 +1534,90 @@
                             "default": true
                         }
                     },
-                    "oneOf": [
+                    "allOf": [
```

Comment:
Maybe it's worth to add some documentation here, why we need it and what it does

Reply:
Yeah, definitely, for this one and many more parts of the schema, especially these more complex parts.

Reply:
For this part here it's a bit harder to have "inline" documentation that explains if/then logic, because not all JSON schema objects can have description, so we can explain this on a higher level in cluster chart docs.